third_party/cpgfunctionEP-0.3.2/include/cpgfunction/gfunction.h

```diff
@@ -0,0 +1,63 @@
+// -*- lsst-c++ -*-
```

Comment:
Functions related to the g-function calculation methodology described in Cimmino and Bernier (2014).